### DISCUSSION TITLE
训练到多少步才算好？

### DISCUSSION THREAD

**Discussion by yanliusama** [svc-develop-team/so-vits-svc#346] [General]
我拥有30分钟左右的训练集，我试着训练了1600步和48000步，转换出来的歌声都没有达到我想要的效果

**Comment by popcorn333** [comment]
我也是 我训练了1k epoch 但我没用diffusion network

**Comment by HoshikawaYoru** [comment]
一般情况下50000+步就差不多了，100000+步差不多就完美了
